Update Engineering Reference to reflect current Fanger PMV model

RKStrand opened the pull request:
Addresses #4341 
Changes made to the Engineering Reference to reflect changes that have
been made to the Fanger model code

RKStrand committed:
Merge branch 'develop' into 113837367-Fanger-PMV-Equation-Bug

RKStrand committed:
Merge branch 'develop' into 113837367-Fanger-PMV-Equation-Bug

RKStrand committed:
Modifications to address changes to code

Changes made to the Engineering Reference to reflect changes that have
been made to the Fanger model code

rraustad commented on doc/src/docs/EngineeringReference/13f-EncyclopaedicRefs.md:
```diff
@@ -544,16 +544,12 @@ EvapHeatLossRegComf = 0.42\*(IntHeatProd - ActLevelConv)
-0.68 is the passive water vapor diffusion rate, (g/h·m<sup>2</sup>·Torr)
-
-0.61 is the latent heat of water, (W·h/g)
-
```
I see nothing else wrong between docs and code except for the following:

The remaining definitions below the "Where:" at this point are also not used and should be deleted (i.e., Psk, Psk= 1.92..., SatSkinVapPress = ..., Tskr = ..., and SkinTempComf), as should the "Where:". I also question the following paragraph. Does the model still calculate skin temperature? I don't see it except in comment lines. I also wonder if the new model still relies on thermal sensation votes from subjects at KSU and Denmark? Revise the paragraph wording to simply state the obvious? If you agree, then the entries in Table 81 that were deleted should also be removed (probably harder to do than it should be).

I could delete the "Where:" block and revise the following paragraph, but I an not familiar enough with the recent changes so I ask these questions instead.

rraustad commented on doc/src/docs/EngineeringReference/13f-EncyclopaedicRefs.md:
```diff
@@ -544,16 +544,12 @@ EvapHeatLossRegComf = 0.42\*(IntHeatProd - ActLevelConv)
-
-0.61 is the latent heat of water, (W·h/g)
-
 P<sub>sk</sub> is the saturated water vapor pressure at the skin temperature required to achieve the thermal comfort
```
Ahh, delete from <tr> to </tr> to delete a row in the table. That should make it pretty easy to do.

RKStrand committed:
Merge branch 'develop' into 113837367-Fanger-PMV-Equation-Bug

RKStrand committed:
Mods to address review comments

Modifications were made to the file to address review comments by Rich
Raustad of FSEC

rraustad commented:
@Myoldmopar @mjwitte I'm OK with these changes. Comments addressed. Review complete.
